homeassistant/components/command_line/notify.py

```diff
@@ -21,21 +21,24 @@
 )
 
 
+# pylint: disable=unused-argument
 def get_service(hass, config, discovery_info=None):
     """Get the Command Line notification service."""
     command = config[CONF_COMMAND]
     timeout = config[CONF_COMMAND_TIMEOUT]
+    name = config[CONF_NAME]
```

Comment:
Name is optional, thus doesn't have to be set.

Reply:
Hm right. And I guess if its not set then there's no way for me to know the name. The documentation says it defaults to `notify` but its really just set by hass right? If there are multiple nameless notifiers then they'll probably be `notify`, `notify1`, `notify2`, etc. and I can't predict that, right?

I think the best I can do here is change it to use `config.get(CONF_NAME)` and default to `''` if that doesn't exist. That means the debug log won't have a name in it but if they didn't give me one then that's really all I can do.